src/sentry/static/sentry/app/views/organizationDashboard/platformList.jsx

```diff
@@ -37,12 +42,7 @@ export default class PlatformList extends React.Component {
     return (
       <Flex align="center">
         <div className="org-dashboard-platform-list">{this.getIcons(platforms)}</div>
-        <PlatformText>
-          {platforms
-            .slice()
-            .reverse()
-            .join(', ')}
-        </PlatformText>
+        <PlatformText>{platforms.join(', ')}</PlatformText>
```

Comment:
Do we still want `PlatformText` to have `overflowEllipsis`

Reply:
I think it's ok if it spans 2 lines for now